diff --git a/magick/blob.c b/magick/blob.c
index 246f85c..d2e5902 100644
--- a/magick/blob.c
+++ b/magick/blob.c
@@ -518,7 +518,7 @@ MagickExport MagickBooleanType CloseBlob(Image *image)
     case PipeStream:
     {
       if (image->blob->synchronize != MagickFalse)
-        status=fsync(fileno(image->blob->file_info.file));
+        status=NULL; // fsync(fileno(image->blob->file_info.file));
       status=ferror(image->blob->file_info.file);
       break;
     }
@@ -543,7 +543,7 @@ MagickExport MagickBooleanType CloseBlob(Image *image)
       if ((image->blob->file_info.file != (FILE *) NULL) &&
           (image->blob->synchronize != MagickFalse))
         {
-          (void) fsync(fileno(image->blob->file_info.file));
+          // (void) fsync(fileno(image->blob->file_info.file));
           status=ferror(image->blob->file_info.file);
         }
       break;
diff --git a/configure b/configure
index afbceb2..03f3f6a 100755
--- a/configure
+++ b/configure
@@ -26258,7 +26258,7 @@ fi
 
 # Assume that delegate headers reside under same directory as ImageMagick
 # installation prefix.
-MAGICK_CPPFLAGS="-I$INCLUDE_DIR/${PACKAGE_NAME}-$MAGICK_MAJOR_VERSION $MAGICK_CPPFLAGS"
+MAGICK_CPPFLAGS="-I$INCLUDE_DIR $MAGICK_CPPFLAGS"
 
 #
 # Find the X11 RGB database
diff --git a/configure b/configure
index 03f3f6a..96db2e8 100755
--- a/configure
+++ b/configure
@@ -32538,7 +32538,7 @@ fi
 #
 
 # Path to ImageMagick header files
-INCLUDE_RELATIVE_PATH="${PACKAGE_NAME}-${MAGICK_MAJOR_VERSION}"
+INCLUDE_RELATIVE_PATH=
 INCLUDE_PATH="${INCLUDE_DIR}/${INCLUDE_RELATIVE_PATH}"
 DEFINE_INCLUDE_PATH="${INCLUDE_DIR}/${INCLUDE_RELATIVE_PATH}/"
 case "${build_os}" in
